wls-gui-wahllokalsystem/src/components/wahlhandlung/beanstandeteWahlbriefe/TheBeanstandeteWahlbriefeTable.vue

```diff
@@ -303,6 +300,10 @@ function _isInputDisabled(rowIndex: number) {
 
 <style scoped>
 td {
-  text-align: center;
+  text-align: start;
+}
+
+.pseudoHeightThatAllowsCellsToGrowInHeight {
+  height: 1px; /* see 2085 */
```

Comment:
nur mit der nummer ist mir nicht sofort klar gewesen dass es das issue sein sollte
```suggestion
  height: 1px; /* see #2085 */
```